src/components/KeyboardAvoidingView/index.android.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View, KeyboardAvoidingView as KeyboardAvoidingViewComponent} from 'react-native';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** In most cases, we do not need to use KeyboardAvoidingView on Android, so it is set to false by default. */
+    isApplyToAndroid: PropTypes.bool,
```

Comment:
Let's rename this to `shouldApplyToAndroid`

Reply:
👍🏻